contract/contracts/OTC.sol

```diff
@@ -0,0 +1,206 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "./libraries/Config.sol";
+import "./externals/IDC.sol";
+import "./interfaces/IOfferFactory.sol";
+import "./interfaces/IOffer.sol";
+
+// Uncomment this line to use console.log
+// import "hardhat/console.sol";
+
+contract OTC is Ownable {
```

Comment:
let's have some admin-controlled upper limit for each asset type. I want to control the potential damage in case there are bugs

Reply:
Deposit upper limit?

Reply:
Total deposit upper limit. Can add later, no rush